tests/conftest.py

```diff
@@ -214,6 +214,11 @@ def __init__(self):
                 "cosine_loss_exponent_tilt": 1.88,
                 "ref_air_density": 1.225,
                 "ref_tilt": 5.0,
+                "helix_a": 1.809,
```

Comment:
These are different from the values in the NREL 5MW file. Should they be consistent?